packages/fluent-theme/src/fluent/styles/Nav.styles.ts

```diff
@@ -0,0 +1,48 @@
+import { INavStyleProps, INavStyles } from 'office-ui-fabric-react/lib/Nav';
+
+export const NavStyles = (props: INavStyleProps): Partial<INavStyles> => {
+  const { theme, isDisabled, isSelected, isGroup, isLink, navHeight = 44 } = props;
+  const { semanticColors } = theme;
+
+  return {
+    link: [
+      {
+        height: navHeight,
+        lineHeight: `${navHeight}px`
+      },
+      !isDisabled && {
+        selectors: {
+          '.ms-Nav-compositeLink:hover &': {
```

Comment:
Instead of using a global selector here, can you just create a global classname set and reference that? That way there isn't a dependency on a global class that could change later.

https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/merge-styles/README.md#referencing-child-elements-within-the-mergestylesets-scope